src/NUnitFramework/framework/Internal/DisplayName.cs

```diff
@@ -223,6 +223,22 @@ public static string GetValueString(object? arg, int stringMax)
             return display;
         }
 
+        public static string EscapeControlChars(string s)
```

Comment:
@manfred-brands Was this still needed? I don't see it referenced

Reply:
@stevenaw I added it initially to allow escaping characters for all TestNames. Similar like what is done in case the parameter is a string. As we haven't decided on an approach, I added a commit to remove it.
That allows this branch to be merged and 4.2 to be released.

Reply:
Thanks @manfred-brands . Definitely nice to get this merged in case it's blocking others from upgrading